docs/new_horizen/05-smart_contract/02-smart_contract_functionality.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Smart Contract Functionality
+---
+
+# Smart Contract Functionality
+
+## Overview
```

Comment:
The tech spec might have been updated since you created this.

We can copy this section directly https://docs.google.com/document/d/1etXcTaFJGmgQd-O6xrfwWa-2a9QUreTftKPJRmfx7ZY/edit#heading=h.94hkyqzeykuk

Reply:
thank you I did not notice this!